tests/lib/Validator/Constraint/LocationIsWithinCopySubtreeLimitValidatorTest.php

```diff
@@ -24,17 +24,13 @@ class LocationIsWithinCopySubtreeLimitValidatorTest extends TestCase
 {
     private const COPY_LIMIT = 10;
 
-    /** @var \Ibexa\Contracts\Core\Repository\LocationService|\PHPUnit\Framework\MockObject\MockObject */
-    private $locationService;
+    private LocationService&MockObject $locationService;
```

Comment:
```suggestion
    private LocationService & MockObject $locationService;
```
It should be like this according to our new fixer rules. Was there a problem with package release?

Reply:
First we will finish Syfmony 7 upgrade and after that we will take care of code style.